olive/workflows/run/run.py

```diff
@@ -153,6 +153,10 @@ def run_engine(config: RunConfig, data_root: str = None):
         config.engine.azureml_client_config = config.azureml_client
 
     engine = config.engine.create_engine()
+
+    # config file will be uploaded to AML job
```

Comment:
could we make this change to aml specific instead of change all the system and engine APIs?

Reply:
I don't think we can skip engine as host/target is only initialized when engine runs. I can try to set config file to aml system directly through engine without changing other systems.

Reply:
when creating system, we can set the config for AMLSystem